data/json/items/melee.json

```diff
@@ -2283,5 +2302,59 @@
     "techniques": "RAPID",
     "qualities": [ [ "CUT", 1 ], [ "CUT_FINE", 1 ], [ "BUTCHER", 7 ] ],
     "flags": [ "STAB", "SHEATH_KNIFE" ]
+  },
+  {
+    "id": "ji",
+    "type": "GENERIC",
+    "name": "ji",
+    "name_plural": "ji",
+    "//": "Couldn't find a good way to pluralize the transliteration.",
+    "description": "This is a bronze polearm that originated in the Shang dynasty of China, if not earlier.  It combines a spear head with the perpendicular blade of the earlier ge or dagger-axe.",
+    "weight": 3175,
+    "volume": 15,
+    "price": 50000,
+    "to_hit": 2,
+    "bashing": 16,
+    "cutting": 31,
+    "material": [ "wood", "bronze" ],
+    "symbol": "/",
+    "color": "yellow",
+    "techniques": [ "WBLOCK_1", "DEF_DISARM" ],
+    "flags": [ "DURABLE_MELEE", "REACH_ATTACK" ]
+  },
+  {
+    "id": "spear_dory",
+    "type": "GENERIC",
+    "name": "dory",
+    "name_plural": "dories",
+    "description": "A well-made spear with a bronze head, Greek in origin.",
+    "weight": 1598,
+    "volume": 14,
+    "price": 10000,
+    "to_hit": 2,
+    "bashing": 8,
+    "cutting": 28,
+    "material": [ "bronze", "wood" ],
+    "symbol": "/",
+    "color": "yellow",
+    "techniques": [ "WBLOCK_1", "IMPALE" ],
+    "qualities": [ [ "COOK", 1 ] ],
+    "flags": [ "SPEAR", "REACH_ATTACK", "DURABLE_MELEE" ]
+  },
+  {
+    "id": "sword_forged",
```

Comment:
This seems redundant with the arming sword, what differentiates them?

Reply:
Material of make, but eh, it's out of place, you're right. I'll get rid of it. Replacing with xiphos where appropriate.